src/docs/product/relay/operating-guidelines.mdx

```diff
@@ -21,64 +21,12 @@ This page reviews our guidelines for operation when self-hosting external Relays
-          </li>
-        </ul>
-      </td>
-    </tr>
-    <tr>
-      <td>Storage</td>
-      <td>
-        <ul>
-          <li>Relay does not currently require disk storage.</li>
-        </ul>
-      </td>
-    </tr>
-  </tbody>
-</table>
+| Resource | Recommendations                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU      | - **Required**: x86-64 (amd64) CPU architecture<br/> - Multi-core CPU [[1]](#multi-core-cpu)                                                                                                                                                                                                                                                                                                                                                          |
+| Memory   | - At least 2GB of RAM per Relay container is recommended.                                                                                                                                                                                                                                                                                                                                                                                             |
+| Network  | - Bandwidth: Make sure you have enough capacity to receive and forward the planned data volume. Relay compresses all upstream requests to `sentry.io` by default, but the compression rate might vary depending on the types and shapes of the submitted events.<br/> - Latency: Relay can tolerate network delays up to a certain point. It is however recommended to make sure that the round-trip time to the upstream stays lower than 5 seconds. |
+| Storage  | - By default Relay does not require the disk storage.<br/> - The storage is required only if [Spooling](/product/relay/options/#spooling) is configured, see `spool.envelopes.max_memory_size` configuration option.                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| Storage  | - Disk storage is only required if [Spooling](/product/relay/options/#spooling) is configured. See the `spool.envelopes.max_memory_size` configuration option. Otherwise, Relay does not require disk storage.                                                                                                                                                                                                                               |
```